### PR TITLE
Cleaned up docblocks and applied PSR-2 formatting

### PR DESCRIPTION
**Pull request by Jasrags**
Vance, First off thank you for spot, Jake pointed me to it and i have been using it for my new project. I was noticing that the docblocks and formatting could use a little love and was wondering if you are cool with this kind of pull request before i went over the other files. 

The docblock cleanup helps me as PHPstorm uses them for code inspection and some are were off or just missing. 

I left some things i did not know empty in the docblocks like the _noQuote param but i put in the placeholder.

Thoughts?

### CODE REVIEW

File: lib/Spot/Query.php [hunk 1]
```diff
@@ -9,26 +9,61 @@
...
+     */
     protected $_tableName;
+
+    /**
+     * @var
+     */
     protected $_queryBuilder;
+
+    /**
+     * @var
```
Comment (vlucas) [inline lib/Spot/Query.php]: Should be `boolean`

File: lib/Spot/Query.php [hunk 2]
```diff
@@ -9,26 +9,61 @@
...
+     */
     protected $_noQuote;
 
-    // Storage for query properties
+    /**
+     * Storage for query properties
```
Comment (vlucas) [inline lib/Spot/Query.php]: Change to "Storage for eager-loaded relations"

**Comment by vlucas** [comment]
Thanks! I love any and all pull requests that improve the project, and this one will certainly help a lot of people. I left a few notes to clean up, and then I will merge.

**Comment by Jasrags** [comment]
Changes made :)

**Comment by Jasrags** [comment]
I noticed that in the search function $fieldTypes is never used and '\Spot\Adapter\Mysql' is showing up as an unknown class.

**Comment by vlucas** [comment]
`\Spot\Adapter\Mysql` is an old class from Spot v1. References to it are incorrect, and should be removed.

**Comment by Jasrags** [comment]
What should it be replaced with?

**Comment by Jasrags** [comment]
I think that the removal of \Spot\Adapter\Mysql should be handled in another pull request as this one is all about doc blocks and style.

**Comment by vlucas** [comment]
I agree.

**Pull request merged**